projects/igniteui-angular/src/lib/services/overlay/scroll/absolute-scroll-strategy.ts

```diff
@@ -1,6 +1,9 @@
 import { IgxOverlayService } from '../overlay';
 import { ScrollStrategy } from './scroll-strategy';
 
+/**
+ * Upon scroll scrolls both content and shown component.
```

Comment:
It reacts to scroll, doesn't do any scrolling.
```suggestion
 * On scroll reposition the overlay content.
```